stack/values.yaml

```diff
@@ -73,6 +81,7 @@ kube-prometheus-stack:
   # This prometheus is monitoring the cluster. Do not use it for load generation.
   prometheus:
     prometheusSpec:
+      remoteWriteDashboards: true
```

Comment:
Since we are using this in our monitoring cluster, let's enable this in tobs instead.

Reply:
Sure, https://github.com/timescale/tobs/pull/634